onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -1361,7 +1361,7 @@ def _where_input_wrangler(
         reason="String padding is not accepted by aten::conv2d",
     ),
     TorchLibOpInfo(
-        "nn.functional.conv3d",
```

Comment:
This changes the tested torch function. Is it expected? what's the guideline for picking which torch function?